src/main/webapp/style.css

```diff
@@ -0,0 +1,47 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  size:80%;
+  
+}
+h1 {
+  background-color: #00a6a6;
+  padding: 5%;
+  text-align: center;
+  font-family: Helvetica;
+  color: white; 
+  font-size:1000%;
+}
+h3 {  
+  text-align: center;
+  font-family: Helvetica;
+  font-size:700%;
+}
+
+p {
+  text-align: center;
+    font-size:400%;
+}
+
+
+input[type=button] {
```

Comment:
You may want to have a class for this instead. Not all buttons might end up being the same in the final version (ex. cancel buttons might be red or differently shaped). Classes use # at the start. Same goes for the img tag below.

Reply:
when I make "input[type=button]" a class, it doesn't apply to the buttons.

Reply:
Check out this link:

https://www.w3schools.com/howto/howto_js_add_class.asp

You'll need to add the class to any button you want to have that style. So in CSS it'd have something like this:

```
#the-class-name {
   font-size: 100%;
}
```

And in javascript you'd have this:

```
var element = document.getElementById("the-button");
element.classList.add("the-class-name");
```

If you're creating the button in the code itself, you don't need to get the element by ID.